.github/workflows/build.yml

```diff
@@ -1,15 +1,20 @@
-name: Build
+name: Build and tag nightlies
 
-on: [push]
+on:
+  push:
+    branches:
+      - master
```

Comment:
And only on tag here, no?

Reply:
I'm still kind of working on this... I think on push to dev we'll create the tag and a pre-release, and then we can have a check on master to create a tag and draft up a full release. 

We could also have dev just create tags only, with no release, and point users to the tags as a "bleeding edge" link, with new features from the dev branch?

Reply:
As talked in the Discord, this will draft up a full release for us, so we just have to verify the release notes before publishing :)